data/coalition.txt

```diff
@@ -735,3 +735,151 @@ phrase "friendly heliarch"
 		"we have committed."
 		"we do serve."
 		"we do vow."
```

Comment:
this isn't really part of your PR, but could you also split the period here to another word?

Reply:
I thought it is not needed to separate periods to another word because of the current hails. I think we need a standard for writing hails, for the sake of consistency.

Reply:
the standard is basically splitting the interpunction at the end of a hail and spaces between the "words" in a hail
while your PR just adds some hails, you could standardize the whole file

Reply:
I am wondering, is the standard de facto, or said by MZ himself?

Reply:
not sure if it was ever clearly said, but MZ and others have done this before, so it's at least implied to some extend
although while creating whole new files, one may forget some standards at times